### PR TITLE
Update pt.json (Cypher System)

### PR DESCRIPTION
**Pull request by lfas**
Translation based on the official release in Brazil by New Order.

### CODE REVIEW

**Comment by Anduh** [comment]
Your PR have apparently some conflict you need to solve before this can be accepted.

**Comment by Anduh** [comment]
Seems to be fine now, good job!

File: Cypher Systems by Roll20/translations/pt.json [hunk 1]
```diff
@@ -1,247 +1,247 @@
...
+    "untrained-abbreviation": "N",
     "trained-abbreviation": "T",
-    "specialized-abbreviation": "S",
-    "used": "Used",
-    "simple-font": "Simple Font",
-    "limit-subsection-height": "Limit subsections height"
+    "specialized-abbreviation": "E",
+    "used": "Usado"
+    "simple-font": "Fonte simples",
+    "limit-subsection-height": "Limitar altura das sub-seções"
```
Comment (lfas) [inline Cypher Systems by Roll20/translations/pt.json]:
```suggestion
    "limit-subsection-height": "Limitar altura das subseções"
```

**Comment by clevett** [comment]
@lfas 

If email me at cassie@roll20.net  I can send you an invite to Crowdin. Crowin is the website we use for translating Roll20 character sheets and the website. I find it a bit easier to use than translating JSON files. :)

Thanks,
Cassie

**Pull request merged**